src/main/java/com/example/parser/utils/Parser.java

```diff
@@ -39,46 +41,34 @@ public void parser(String filePath) throws IOException, ParseException {
 
         Iterator i = jsonArray.iterator();
 
-        if(BD_list.size() == 0){
-            while (i.hasNext()) {
-                JSONObject innerObj = (JSONObject) i.next();
-                permit = new PermitForEmissionsOfPollutants();
-
-                permit.setEdrpou(String.valueOf(innerObj.get("edrpou")));
-                permit.setNumber(String.valueOf(innerObj.get("number")));
-                permit.setName(String.valueOf(innerObj.get("name")));
-                permit.setData_type(String.valueOf(innerObj.get("data_type")));
-                permit.setValidity(String.valueOf(innerObj.get("validity")));
-                permit.setLegal_address(String.valueOf(innerObj.get("legal_address")));
-                permit.setActual_address(String.valueOf(innerObj.get("actual_address")));
-                permitRepository.save(permit);
-
-                JsonList.add(permit);
-                File_counter++;
-            }
+        while (i.hasNext()) {
+            JSONObject innerObj = (JSONObject) i.next();
+            permit = new PermitForEmissionsOfPollutants();
+
+            permit.setEdrpou(String.valueOf(innerObj.get("edrpou")));
+            permit.setNumber(String.valueOf(innerObj.get("number")));
+            permit.setName(String.valueOf(innerObj.get("name")));
+            permit.setData_type(String.valueOf(innerObj.get("data_type")));
+            permit.setValidity(String.valueOf(innerObj.get("validity")));
+            permit.setLegal_address(String.valueOf(innerObj.get("legal_address")));
+            permit.setActual_address(String.valueOf(innerObj.get("actual_address")));
+            JsonList.add(permit);
+            File_counter++;
         }
 
-//        permit.setEdrpou("1");
-//        permit.setNumber("1");
-//        permit.setName("1");
-//        permit.setData_type("1");
-//        permit.setValidity("1");
-//        permit.setLegal_address("1");
-//        permit.setActual_address("1");
-//
-//        JsonList.add(permit);
-
         Set<PermitForEmissionsOfPollutants> union = new HashSet(JsonList);
         union.removeAll(BD_list);
 
-        for (PermitForEmissionsOfPollutants k : union)
+        for (PermitForEmissionsOfPollutants k : union) {
+            permitRepository.save(k);
             DB_counter++;
```

Comment:
Variable name should start from the lower case and without underscore

Reply:
done